test/lib/utils/service/systemd_test.rb

```diff
@@ -109,6 +109,10 @@ module ForemanMaintain
       it 'can compare with other obejcts' do
         _(httpd_service.matches?(nil)).must_equal false
       end
+
+      it 'can compare by wildcard service' do
+        _(httpd_service.matches?('httpd@*.service')).must_equal true
```

Comment:
```suggestion
        _(httpd_service.matches?('http*.service')).must_equal true
```
there is no `httpd@` service defined, only `httpd`